README.md

```diff
@@ -1,7 +1,7 @@
-<p align="center">
+<div align="center">
```

Comment:
The rich diff of GitHub shows that this will not make it centered anymore. I think we encountered this issue before as well which is why we're using `<p>` instead of `<div>`

Reply:
I'll change it to only use p and h1, but we can't put a h1 inside of a p 😅

Reply:
Fixed